examples/cosmos-sdk/transfer/transfer.md

```diff
@@ -27,23 +30,39 @@ pip install --upgrade poetry
 
 ```
 brew install pyenv git go java
+```
+
+Because `macOS` does not come with `python3.10.5`, we are using `pyenv` install and managed between different versions of `python`.
```

Comment:
Why is `pyenv` used only for Mac? Is it not the case that Linux too will not have by default installed python 3.10? 

Would it make sense to factor out this comment and mention it as a general way to manage different versions of python?

Reply:
By Linux, do you mean Ubuntu? Latest Ubuntu has 3.10 as the default Python3.

But for some other Linux distros, it may not be true, e.g. Debian.